ext/odbc/tests/odbc_primarykeys_001.phpt

```diff
@@ -0,0 +1,37 @@
+--TEST--
+odbc_primarykeys(): Basic test for odbc_primarykeys()
+--EXTENSIONS--
+odbc
+--SKIPIF--
+<?php include 'skipif.inc'; ?>
+--XFAIL--
+Doesn't work with MS SQL
+--FILE--
```

Comment:
Improve skipif to get rid of XFAIL?

Reply:
But also seems like it should work on MS SQL according to: https://www.php.net/manual/en/function.odbc-primarykeys.php#40061

Reply:
Yes, but I'm not able to retrieve a meaningful result, no matter what combination of parameters I provide :(